src/braket/pennylane_plugin/translation.py

```diff
@@ -12,42 +12,58 @@
 # language governing permissions and limitations under the License.
 
 from functools import singledispatch
-from typing import FrozenSet, Type
+from typing import FrozenSet
 
+import numpy as np
 import pennylane as qml
 from braket.circuits import Gate, ResultType, gates, observables
 from braket.circuits.result_types import Expectation, Probability, Sample, Variance
 from pennylane.operation import Observable, ObservableReturnTypes, Operation
 
+from braket.pennylane_plugin.ops import (
+    ISWAP,
+    PSWAP,
+    XX,
+    XY,
+    YY,
+    ZZ,
+    CPhaseShift,
+    CPhaseShift00,
+    CPhaseShift01,
+    CPhaseShift10,
+)
+
 _OPERATION_MAP = {
```

Comment:
This shouldn't be needed anymore, since all of the translations are in the singledispatch.